lib/glm/CMakeLists.txt

```diff
@@ -6,4 +6,4 @@ set_target_properties(glm PROPERTIES FOLDER "3rdparty")
 
 # Disable warnings in this library
 suppress_warnings(glm)
-target_compile_definitions(glm PUBLIC GLM_FORCE_CXX11 GLM_FORCE_RADIANS)
\ No newline at end of file
+target_compile_definitions(glm PUBLIC GLM_FORCE_RADIANS)

```

Comment:
Did we create this CMakeLists.txt or was it provided by them?  If it was part of their bundle, is some other place to override it outside of their provided code?  If we made it, ignore this :)

Reply:
That's a file I wrote. Normally you're supposed to define that before every usage of a GLM header but a global define is a better option here.